docs/devguide/dds_security.rst

```diff
@@ -72,19 +72,23 @@ As of this writing, vcpkg is only supported on Visual Studio 2015 Update 3 and l
 
        * Fetch and build the dependencies; by default, vcpkg targets x86 so be sure to specify the x64 target if required by specifying it when invoking vcpkg install, as shown here:
 
-         ``vcpkg install openssl:x64-windows xerces-c:x64-windows``
+.. code-block:: doscon
+
+         vcpkg install openssl:x64-windows xerces-c:x64-windows
```

Comment:
This should probably line up with the start of "Fetch and build", as it's [kinda messing up the next list item](https://opendds--4102.org.readthedocs.build/en/4102/devguide/dds_security.html#building-opendds-with-security-on-windows), but there's a problem with this whole section with the `* * *` at the top. This is a place where the conversion script was confused and the two (or 3? not actually sure) extra asterisks should be removed and everything should be realigned.